frontend/src/metabase/nav/containers/MainNavbar/MainNavbarContainer/MainNavbarView.tsx

```diff
@@ -242,6 +242,18 @@ export function MainNavbarView({
                 role="tree"
                 aria-label="collection-tree"
               />
+              {isAdmin && (
+                <PaddedSidebarLink
+                  icon={
+                    getCollectionIcon(
+                      PERSONAL_COLLECTIONS as Collection,
+                    ) as unknown as IconName
```

Comment:
I don't think you need any of these casts. Also, since we're just hard coding this, you could hard-code the icon name

```
icon="group"
```

Reply:
You're absolutely right!
I left the previous implementation intact without even looking at it. But there's no reason to leave it like this.
Thanks

https://github.com/metabase/metabase/pull/57605/commits/a9ef49890659f666eafdfd025d0946785b5da3ee